### PR TITLE
MB-2108 Important shipment dates component

### PR DESCRIPTION
**Pull request by garrettqmartin8**
## Description

A new component for rendering important shipment dates!

## Setup

`make storybook`

## Code Review Verification Steps

* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-2108) for this change

### CODE REVIEW

File: src/components/Office/importantShipmentDates.module.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
+@import '../../shared/styles/basics';
+@import '../../shared/styles/mixins';
+@import '../../shared/styles/colors';
+
+.container-override {
+  background: $base-lightest;
+  padding: 0 1.6rem 1rem 1.6rem;
+}
```
Comment (gidjin) [inline src/components/Office/importantShipmentDates.module.scss:8]: Is this container supposed to have its own style? Seems like it should be using one of the container examples in storybook under `Components -> Containers`.  I'm guessing you want `.container .container--gray` on the `div` but I'd check with design to be sure.

**Comment by garrettqmartin8** [comment]
@gidjin made some changes - thanks for the feedback!

File: src/components/Office/ImportantShipmentDates.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import * as PropTypes from 'prop-types';
+import styles from './importantShipmentDates.module.scss';
+
+const cx = classNames.bind(styles);
+
+const ImportantShipmentDates = ({ requestedPickupDate, scheduledPickupDate }) => {
+  return (
+    <div className={`container container--gray maxw-tablet ${cx('shipment-dates-container')}`}>
+      <div>
+        <p className={`${cx('header')} font-sans-2xs text-bold text-base-dark display-inline-block margin-bottom-0`}>
```
Comment (tinyels) [inline src/components/Office/ImportantShipmentDates.jsx:12]: It seems a bit weird for this to be a paragraph. Would it be better to use a header element here (or some other semantic markup) to make it clearer to screen readers that this is a header? @transcom/truss-design, what do you recommend?

Comment (garrettqmartin8) [reply src/components/Office/ImportantShipmentDates.jsx]: Good call - I changed this to use `h4` elements, which is hopefully a bit more semantically descriptive.

**Comment by garrettqmartin8** [comment]
@dlee11 made those adjustments - let me know if there's anything else I should address.

**Comment by garrettqmartin8** [comment]
@dlee11 updated!

**Pull request merged**